gtfs-realtime/spec/en/reference.md

```diff
@@ -502,11 +503,32 @@ An internationalized message containing per-language versions of a snippet of te
 
 A localized string mapped to a language.
 
+| _**Field Name**_ | _**Type**_ | _**Required**_ | _**Cardinality**_ | _**Description**_ |
+|------------------|------------|----------------|-------------------|-------------------|
+| **text** | [string](https://developers.google.com/protocol-buffers/docs/proto#scalar) | Required | One | A UTF-8 string containing the message. |
+| **language** | [string](https://developers.google.com/protocol-buffers/docs/proto#scalar) | Conditionally required | One | BCP-47 language code. Can be omitted if the language is unknown or if no internationalization is done at all for the feed. At most one translation is allowed to have an unspecified language tag - if there is more than one translation, the language must be provided. |
```

Comment:
Was this a change that also appeared out of nowhere?

Reply:
This is just the way git is displaying the changes. `language` from Translation is "moved" to `LocalizedImage` and `text` is deleted and re-added. This doesn't affect the content itself. 

There's no way to fix that afaik.

Reply:
@gcamp I _assumed_ that might have been the case. Thanks for confirming.